### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by znicholls**
# Please confirm that this PR has done the following:

- [x] Tests Added (N/A)
- [x] Documentation Added (N/A)
- [x] Description in RELEASE_NOTES.md Added (N/A)

## Adding to RELEASE_NOTES.md (remove section after adding to RELEASE_NOTES.md)

Please add a single line in the release notes similar to the following:

```
- (#XX)[http://link-to-pr.com] Added feature which does something
```

# Description of PR

Fixes syntax in README to make header appear as header (I'd forgotten the file was markdown not rst...)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,8 +73,7 @@ To install from source after cloning this repository, simply run
 pip install -e .
 ```
 
-Development
-+++++++++++
+### Development
```
Comment (danielhuppmann) [inline README.md]: should't this heading be at the same level as the one above (Install)?

Comment (znicholls) [reply README.md]: If you wish, my thought was that it's install instructions for development, rather than just install (which is the heading above)

Comment (danielhuppmann) [reply README.md]: @gidden, what's your take here?

**Comment by gidden** [comment]
lgtm

**Pull request merged**